### PR TITLE
release blog for 0.26

### LINKED ISSUE #4322: Post release 0.26 blog
Publish blog post with release notes for new version 0.26
Make sure to mention the first release of the component https://github.com/knative-sandbox/eventing-kogito

/assign

### PR DESCRIPTION
**Pull request by csantanapr**
release blog for 0.26
Fixes #4322

### CODE REVIEW

File: blog/releases/announcing-knative-v0-26-release.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+    - [Eventing Kogito v0.26](#eventing-kogito-v026)
+- `kn` [CLI v0.26](#Client-v026)
+- [Operator v0.26](#Operator-v026)
+- [Thank you contributors](#Thank-you-contributors)
+
+
+### Highlights
+
+- **Kubernetes 1.20+ is now required**
+- Improvements for istio mesh compatibility mode.
```
Comment (omerbensaadon) [inline blog/releases/announcing-knative-v0-26-release.md:36]:
```suggestion
- Improvements for Istio mesh compatibility mode.
```

**Comment by omerbensaadon** [comment]
left 1 small change

**Comment by csantanapr** [comment]
thank you sir @omerbensaadon 👍

**Pull request merged**